gem/styles/gem-springster/state/comments/_s-comment-counter.scss

```diff
@@ -1,8 +1,10 @@
 .comments__alias {
   font-weight: normal;
 }
-.comment-counter {
-  position: absolute;
-  margin-top: 0;
-  right: 10px;
+.teaser-stacklist__item--content {
+  .comment-counter {
+    position: absolute;
+    margin-top: 10px;
```

Comment:
how is this displaying on devices?

still aligned like this?
<img width="360" alt="screen shot 2017-05-26 at 9 03 20 am" src="https://cloud.githubusercontent.com/assets/9653693/26495535/3bd087fe-4224-11e7-9a10-0a2e2aa73d78.png">